corehq/apps/domain/templates/login_and_password/two_factor/_base.html

```diff
@@ -0,0 +1,7 @@
+{% extends "style/bootstrap2/centered.html" %}
```

Comment:
yeah. i migrated all of the internal forms to b3. the only thing thats b2 is the login page, because its a clone of the old one.